index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Christmas Tree Exercise</title>
+</head>
+<body>
+
+
+	How high do you want your tree? Enter a numerical value from 1 - 10:<br>
+
+	<input type="text" id="tree_height" value="1 - 10"><br>
+
```

Comment:
Instead of setting `value` equal to 1 - 10, try using `placeholder`. That will fill the space, but then disappear when the user starts typing. As `value` the user has to delete what's in the text box first.